README.md

```diff
@@ -135,6 +135,23 @@ sslCipherSuite | The prioritized list of allowed SSL/TLS ciphers. For more infor
 
 Similar to configuring the server, the client configuration file is located at `/etc/opt/omi/conf/omicli.conf`.
 
+### Collecting Performance Metrics using OMI/OMI Provider on docker container or virtual machine
```

Comment:
I will recommend to move this docker related part into the below doc. If you want, you can add the below hyperlink in the main OMI Readme doc. 

https://github.com/Microsoft/omi/blob/master/docker/README.md